### PR TITLE
Added support for RenderMode Cpu

### PR DESCRIPTION
**Pull request by christopherlagerhult**
Includes changes for ~~[link removed]~~ to work.
Review @emilwestergren (and maybe someone else too)

### CODE REVIEW

**Comment by marcusnaslund** [comment]
Edited your post. Please don't link to (or mention) private repos from here - instead link from the other PR to this one. It's safer. :)

File: source/draw/gpu/opengl/backend/gles3/external/gles3.ooc [hunk 1]
```diff
@@ -6,7 +6,6 @@
  * of the MIT license.  See the LICENSE file for details.
  */
 
-version(!gpuOff) {
```
Comment (emilwestergren) [inline source/draw/gpu/opengl/backend/gles3/external/gles3.ooc]: Do you really need to remove this? This kind of code we want inside `!gpuOff`

Comment (christopherlagerhult) [inline source/draw/gpu/opengl/backend/gles3/external/gles3.ooc]: Not if the "dependent" parts are wrapped inside version flags. I will reinstate those flags

Comment (christopherlagerhult) [inline source/draw/gpu/opengl/backend/gles3/external/gles3.ooc]: Updated

**Comment by emilwestergren** [comment]
Much better :)

**Pull request merged**